src/RoundTabler/PCIScan.java

```diff
@@ -31,9 +31,9 @@ public PCIScan(){
 		this.pMYSQLColumnTypes = "(mediumtext, longtext, text, tinytext, varchar)";
 	}
 
-	public PCIScan(Connection MYSqlDBConnection, String TableName  ){
+	public PCIScan(Connections MYSqlDBConnection, String TableName  ){
```

Comment:
I think it is supposed to be Connection not Connections - getting error with Connections - lol

Reply:
on both of the lines changed from Connection to Connections (line 24)

Reply:
I second that, i think its just Connection

Reply:
I changed Connection to Connections as per PR comment

Reply:
I think @zachrussell12 typoed on his comment

Reply:
yes i did thats my mistake

Reply:
No problem. I changed it back